src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -983,6 +1020,11 @@ static void _updateRepeater(TVG_UNUSED LottieGroup* parent, LottieObject** child
     auto repeater= static_cast<LottieRepeater*>(*child);
 
     if (!ctx->repeater) ctx->repeater = new RenderRepeater();
+    auto matrix = ctx->propagator->transform();
+    if (!mathIdentity(static_cast<const Matrix*>(&matrix))) {
```

Comment:
Please confirm if this identity check is really helpful. I'm concerned it just adds unnecessary checks.

Reply:
I wanted to avoid unnecessary mem allocation, but now should be more optimal:
changed to null check:
```
    if (auto matrix = PP(ctx->propagator)->transform()) {
        if (!ctx->repeater->transform) ctx->repeater->transform = (Matrix*)malloc(sizeof(Matrix));
        *ctx->repeater->transform = *matrix;
    } else ctx->repeater->transform = nullptr;
    ```

Reply:
Maybe `Matrix transform` in Repeater (43 line) might be better than `Matrix* transform` if, in most cases, the repeater needs to allocate the transform."